Sources/PixelKit/PixelKit.swift

```diff
@@ -204,7 +205,7 @@ public final class PixelKit {
             case Source.macDMG.rawValue, Source.macStore.rawValue:
                 headers[Header.client] = "macOS"
             default:
-                break
+                headers[Header.client] = "macOS"
```

Comment:
won't it have some side effect?

Reply:
Basically so far is always macOS and we have some NetP etc sources that still want macOS header, so basically all I am doing here is making sure that is it comes from iOS (that for now it is only used for the experiment) it has the iOS/ipadOS header… avoiding changes for anything else that is using it currently.
When we do the full migration we will need to adjust it but we need to consider other Sources as well